cmd/export_ledger_entry_changes_test.go

```diff
@@ -5,9 +5,11 @@ import (
 	"testing"
 )
 
+const coreExecutablePath = "/Users/isaiahturner/Local/stellar-core/src/stellar-core"
```

Comment:
Replace local directories with arbitrary filepaths that'll work for anyone who clones the repo.